readme.md

```diff
@@ -81,7 +101,14 @@ var Panda = function(name, age) {
 ```
 
 Your Answer:
-```text
+```
+Panda.prototype = {
+  eat_bamboo: function(){
+    var self = this;
+    self.num_bamboo_eaten = self.num_bamboo_eaten + 1;
+    return self.num_bamboo_eaten
+  }
+}
 
```

Comment:
:+1: This works! Just remember that assigning to `Panda.prototype` instead of `Panda.prototype.eat_bamboo` overwrites and other properties/methods assigned to the prototype, so the latter is safer practice.